docs/source/development/settings_page.md

```diff
@@ -0,0 +1,20 @@
+# Settings Page
+
+The settings page (viewable here TODO: add link), is an easy-to-use UI for users to edit their auto-archiver settings.
```

Comment:
is this TODO coming in a following PR or can we predict it now?

Reply:
Good catch, this is fixed now